docs/pages/upgrading/cloud-linux.mdx

```diff
@@ -0,0 +1,179 @@
+---
+title: Upgrade Teleport Cloud Clusters on Linux
+description: Provides instructions for upgrading Teleport Cloud clusters that run on Linux servers. 
+---
+
+This guide explains how to upgrade Teleport Cloud clusters running on Linux
+servers. On Teleport Cloud, Auth Service and Proxy Service upgrades are managed
+for you. To keep agents up to date, you can either enroll them in automatic
+upgrades or upgrade them manually using the method you used to install Teleport.
+
+## Prerequisites
+
+- Familiarity with the [Upgrading Compatibility Overview](./overview.mdx) guide,
+  which describes the sequence in which to upgrade components in your cluster.
+
+- A Teleport Enterprise Cloud or Teleport Team account. You can determine the
+  current version of the Auth Service and Proxy Service by running the following
+  command, where `mytenant` is the name of your Teleport Team or Teleport
+  Enterprise Cloud account. This requires the [`jq` CLI
+  tool](https://jqlang.github.io/jq/):
+
+  ```code
+  $ curl -s https://mytenant.teleport.sh/webapi/ping | jq '.server_version'
+  ```
+
+Teleport supports automatic agent updates for systemd-based Linux distributions
+using `apt`, `yum`, and `zypper` package managers. The [Automatic Update
+Architecture](../architecture/agent-update-management.mdx) guide describes
+how agent updating works. Automatic agent upgrades require:
+
+- A Teleport Cloud account that supports automatic upgrades. To determine if
+  your account supports automatic upgrades, run the following command, replacing
+  `mytenant.teleport.sh` with the address of your Teleport Cloud account:
+
+  ```code
+  $ curl -s https://mytenant.teleport.sh/webapi/ping | jq '.automatic_upgrades'
+  true
+  ```
+
+- At least one Teleport Enterprise agent, started via systemd on a distribution
+  using the `apt` or `yum` package managers. 
+
+  So you can access each agent, we assume that each agent runs the Teleport SSH
+  Service in addition to any other services. See the [Deploy Teleport Agents
+  with Terraform](../agents/deploy-agents-terraform.mdx] guide for an example.
+
+## Step 1/2. Find agents to upgrade
+
+To list agents that are not enrolled in automatic upgrades, use the `tctl
+inventory ls` command with the `--upgrader=none` flag. This command also uses
+the `--services` flag to constrain the search to Teleport processes running
+agent services:
+
+```code
+$ tctl inventory ls \
+  --services=node,kube,db,app,windows_desktop,discovery,okta \
+  --upgrader=none
```

Comment:
Why the inconsistency between this and the teleport-kube-agent instructions where we filter out the text output with `awk` instead?

Reply:
I think I used this approach after forgetting I'd used `awk` in the other page. I'll use this one in both pages—it's not as concise as a negated service filter, but at least it doesn't require an external tool